### ISSUE TITLE
Cannot reproduce simple examples, some variables not defined

### ISSUE THREAD

**Issue by berndblasius** [wookay/Poptart.jl#22]
Looks like a very promiing project, but I cannot reproduce the examples.
After `using Poptart.Desktop` some variables are defined (e.g., Window, Application), but for others (e.g., Button, Label) I recieve an error `UndefVarError: Button not defined`

**Comment by wookay** [comment]
what's your Pkg status?
```julia
(@v1.5) pkg> st Poptart CImGui
Status `~/.julia/environments/v1.5/Project.toml`
  [5d785b6c] CImGui v1.78.1 `../../dev/CImGui`
  [197b51f5] Poptart v0.3.1 `~/.julia/dev/Poptart`
```

**Comment by berndblasius** [comment]
```(@v1.5) pkg> st Poptart CImGui
Status `~/.julia/environments/v1.5/Project.toml`
  [5d785b6c] CImGui v1.78.1
  [197b51f5] Poptart v0.2.2```

I'm using Julia 1.5.1 and have this error both on a MacOS and Debian machine

**Comment by berndblasius** [comment]
I just fund that some of the missing variables are under Poptart.Controls

So, if I add `using Poptart.Controls` I can use Button, Label, and Canvas.
But not I receive `UndefVarError: exit_on_esc not defined`

**Comment by wookay** [comment]
please get latest Poptart (v0.3.1). 
```
(@v1.5) pkg> up Poptart
```

**Comment by berndblasius** [comment]
Thanks, solved :-)

Sorry, I didn't realize that I was using a an older version. 
`up Poptart` at first didn't change anything.  But then, I realized that I had SimpleDirectMediaLayer pinned to an old version (because latest version produces an error on my machine) which seemingly forced Poptart to v0.2.2.
So, I removed the old SDL, updated my packages and now everything works fine.

Thanks for the fast response
